README.md

```
@@ -34,114 +34,94 @@
-  "limit": 20,                   // Optional: pagination limit
-  "offset": 0                    // Optional: pagination offset
-}
+- `info` - Get container information
+- `start` - Start a container
+- `stop` - Stop a container
+- `restart` - Restart a container
+- `build` - Build/rebuild a container
+- `logs` - Get container logs
+
+**Natural language examples:**
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)